services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -471,7 +471,6 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     # for passing self-signed certificate to spawned services
-    # TODO: fix these variables once the timeout-minutes: 30 is able to start dynamic services
     DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID: str = ""
     DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME: str = ""
     DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: str = ""
```

Comment:
Seeing the usage you have I think it would make more clear to make them `Optional` and default to `None`. This should not impact your code from what I can tell.
Also regarding `DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME` I would provide a description as commented below.

```suggestion
    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID: Optional[str] = None
    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME: Optional[str] = None
    DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: Optional[str] = Field(None, description"...")
```